custom-connectors/Assently E-Sign/Readme.md

```diff
@@ -0,0 +1,16 @@
+﻿
```

Comment:
@kian-beetroot You can refer below readme.md for reference.

[A Sample readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md)

Reply:
do we need to do any changes on the readme file?